ckanext/canada/tests/test_webforms.py

```diff
@@ -224,6 +250,15 @@ def test_new_user_missing_fields(self, app):
         assert 'Password: Please enter both passwords' in response.body
 
 
+    @mock.patch('os.path.isfile', _mock_isfile)
+    @mock.patch('__builtin__.open', _mock_open_ip_list)
+    def test_new_user_bad_ip_address(self, app):
+        offset = h.url_for('user.register')
+        response = app.get(offset, extra_environ=self.extra_environ_tester_bad_ip)
+
+        assert response.status_code == 423
```

Comment:
@RabiaSajjad as discussed, the 423 is not really Nginx: https://developer.mozilla.org/en-US/docs/Web/HTTP/Status/423

So we can just use 403 and condition the templates better.